redis/connection.py

```diff
@@ -1371,7 +1381,13 @@ def _checkpid(self):
             finally:
                 self._fork_lock.release()
 
-    def get_connection(self, command_name, *keys, **options):
+    def get_connection(
+        self,
+        command_name: str,
+        # TODO: Type the next two parameters.
+        *keys,
+        **options
+    ) -> Connection:
```

Comment:
I couldn't figure out what types the `*keys` and `**options` parameters take. If anyone knows please tell me.

Reply:
https://google.github.io/pytype/faq.html#how-do-i-annotate-args-and-kwargs

Reply:
> https://google.github.io/pytype/faq.html#how-do-i-annotate-args-and-kwargs

I know how to type `*args` and `**kwargs`, the question was what do these two parameters take (only integers? Strings? etc.)

Reply:
The right thing to do is set a minimum of:
*keys: List
**options: Dict

Those are accurate typings. It doesn't help the sub options of the various items - but it's realistic.

Reply:
@chayim For `*args` & `**kwargs`, the correct way to type them is to specify the inner type, not List or Dict (that is assumed). See the link I shared.